keadm/cmd/keadm/app/cmd/util/rpminstaller.go

```diff
@@ -51,13 +56,24 @@ func (r *RpmOS) InstallMQTT() error {
 		return nil
 	}
 
-	// install MQTT
-	for _, command := range []string{
+	commands := []string{
 		"yum -y install epel-release",
 		"yum -y install mosquitto",
 		"systemctl start mosquitto",
 		"systemctl enable mosquitto",
-	} {
+	}
+
+	vendorName, err := getOSVendorName()
+	if err != nil {
+		return err
+	}
```

Comment:
Is there no need to throw an error here?
If you can't get it, execute it in the established way.
However, log output is necessary.

Reply:
use fmt.Println to print error

Reply:
what's your opinion? @fisherxu

Reply:
already print error here